mne/preprocessing/xdawn.py

```diff
@@ -111,7 +107,8 @@ def _construct_signal_from_epochs(epochs):
     return data
 
 
-def least_square_evoked(epochs, return_toeplitz=False):
+def least_square_evoked(epochs_data, events, event_id, tmin, tmax, sfreq,
```

Comment:
Fine with me too

On Mon, 11 Jul 2016 at 19:59, Alexandre Gramfort <notifications@github.com>
wrote:

> In mne/preprocessing/xdawn.py
> <https://github.com/mne-tools/mne-python/pull/3245#discussion_r70316324>:
>
> > @@ -111,7 +107,8 @@ def _construct_signal_from_epochs(epochs):
> >      return data
> >
> >
> > -def least_square_evoked(epochs, return_toeplitz=False):
> > +def least_square_evoked(epochs_data, events, event_id, tmin, tmax, sfreq,
>
> ok
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/mne-tools/mne-python/pull/3245/files/c74187460095b9e270108ded30f4d4516e49fc16#r70316324>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AB0fim2q9ke8I8s4Ae5ras-Ae0CU_fnXks5qUpKbgaJpZM4Ij7PZ>
> .
>